### PR TITLE
 zh_CN:quantization.md

### PR DESCRIPTION
**Pull request by kjlintong**
请大佬审校

### CODE REVIEW

File: site/zh-cn/lite/g3doc/convert/quantization.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# 转换量化模型
+本文提供有关如何转换量化 TensorFlow Lite 模型的信息。详细信息，请参阅 [模型优化](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/performance/model_optimization.md)。
```
Comment (Mr-Linus) [inline site/zh-cn/lite/g3doc/convert/quantization.md]:
```suggestion
本文提供有关如何转换量化 TensorFlow Lite 模型的信息。详细信息，请参阅[模型优化](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/performance/model_optimization.md)。
```

File: site/zh-cn/lite/g3doc/convert/quantization.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+# 训练后：针对特定 CPU 型号的量化模型
+创建小模型的最简单方法是在推理期间将权重量化为 8 位并“在运行中”量化输入/激活。这具有延迟优势，但优先考虑减小尺寸。
+
+在转换期间，将optimizations标志设置为针对大小进行优化：
```
Comment (Mr-Linus) [inline site/zh-cn/lite/g3doc/convert/quantization.md]:
```suggestion
在转换期间，将 optimizations 标志设置为针对大小进行优化：
```

File: site/zh-cn/lite/g3doc/convert/quantization.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+转换图表：
+```
+converter = tf.lite.TFLiteConverter.from_saved_model(saved_model_dir)
+converter.inference_type = tf.lite.constants.QUANTIZED_UINT8
+input_arrays = converter.get_input_arrays()
+converter.quantized_input_stats = {input_arrays[0] : (0., 1.)}  # mean, std_dev
+tflite_model = converter.convert()
+```
+对于全整数模型，输入为 uint8。 mean 和 std_dev values 指定在训练模型时这些 UINT8 的值是如何值映射到输入的浮点值。
```
Comment (Mr-Linus) [inline site/zh-cn/lite/g3doc/convert/quantization.md]:
```suggestion
对于全整数模型，输入为 uint8。mean 和 std_dev values 指定在训练模型时这些 UINT8 的值是如何值映射到输入的浮点值。
```

File: site/zh-cn/lite/g3doc/convert/quantization.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+mean 是0到255之间的整数值，映射到浮点数 0.0f。std_dev = 255 /（float_max - float_min）
```
Comment (Mr-Linus) [inline site/zh-cn/lite/g3doc/convert/quantization.md]:
```suggestion
mean 是 0 到 255 之间的整数值，映射到浮点数 0.0f。std_dev = 255 /（float_max - float_min）
```

**Comment by Mr-Linus** [comment]
@kjlintong Update CLAs.Please comment the Pr and let the @googlebot check CLA agagin.

**Comment by kjlintong** [comment]
@Mr-Linus I have signed the CLA .But I don`t know how to update it.What I need to do?

**Comment by lamberta** [comment]
I manually checked the CLA, it's fine.
There's a few commits in this pull request that have multiple authors and that confuses CLAbot 🤖
I can get it through after review. Thanks

**Comment by kjlintong** [comment]
@Mr-Linus @lamberta  Thanks!

**Pull request merged**